nmtwizard/preprocess/consumer.py

```diff
@@ -462,6 +462,8 @@ def _consume(self, outputs):
         tgt_path = os.path.join(self._result_dir, basename + "." + self._tgt_suffix)
         align_path = os.path.join(self._result_dir, basename + ".align")
         write_alignment = meta.get('write_alignment', False)
+        example_weights_path = os.path.join(self._result_dir, basename + ".weights")
+        oversample = meta.get('oversample', False)
```

Comment:
Can you rename the meta field and variable to "example_weights"? We don't need to assume here that the weights are used for oversampling.